### ISSUE TITLE
`cabal install yst` fails

### ISSUE THREAD

**Issue by sehqlr** [jgm/yst#42]
I'm sure that this is my fault, but I've never installed anything through cabal before, and when I run `cabal install yst` I get this message:
```
Resolving dependencies...
cabal: Could not resolve dependencies:
trying: base-4.9.0.0/installed-4.9... (dependency of yst-0.5.1)
next goal: pandoc (dependency of yst-0.5.1)
rejecting: pandoc-1.17.1, pandoc-1.17.0.3, pandoc-1.17.0.2, pandoc-1.17.0.1,
pandoc-1.17 (conflict: yst => pandoc>=1.10 && <1.17)
trying: pandoc-1.16.0.2
next goal: haddock-library (dependency of pandoc-1.16.0.2)
rejecting: haddock-library-1.4.2, haddock-library-1.4.1 (conflict: pandoc =>
haddock-library>=1.1 && <1.3)
rejecting: haddock-library-1.2.1, haddock-library-1.2.0 (conflict:
base==4.9.0.0/installed-4.9..., haddock-library => base>=4.5 && <4.9)
rejecting: haddock-library-1.1.1 (conflict: base==4.9.0.0/installed-4.9...,
haddock-library => base>=4.3 && <4.8)
rejecting: haddock-library-1.1.0 (conflict: base==4.9.0.0/installed-4.9...,
haddock-library => base>=4.5 && <4.8)
rejecting: haddock-library-1.0.1, haddock-library-1.0.0 (conflict: pandoc =>
haddock-library>=1.1 && <1.3)
Backjump limit reached (currently 2000, change with --max-backjumps or try to
run with --reorder-goals).
```
I get the same kind of message if I run it with `--reorder-goals`. If I increase the backjumps to infinity (by setting it to -1) it just hangs. 

What do I need to do? I've tried this with the version of cabal-install that comes from brew, and the full haskell-package distribution from brew cask. I'm on Mac OSX El Capitan.

Thanks!

**Comment by jgm** [comment]
I just released 0.5.1.1 which should be compatible with
the latest pandoc. Try 'cabal update' and then try again.

+++ Sam Hatfield [Jul 07 16 21:00 ]:

**Comment by sehqlr** [comment]
That fixed it! Thanks, I’ll close this now
